src/prettierRule.ts

```diff
@@ -7,10 +7,21 @@ import * as ts from 'typescript';
 
 export class Rule extends tslint.Rules.AbstractRule {
   public apply(sourceFile: ts.SourceFile): tslint.RuleFailure[] {
+    if (this.sourceFileIsIgnored(sourceFile, this.ruleArguments)) {
+      return [];
+    }
     return this.applyWithWalker(
       new Walker(sourceFile, this.ruleName, this.ruleArguments),
     );
   }
+  private sourceFileIsIgnored(sf: ts.SourceFile, ruleArguments: any[]) {
+    let ignorePath = '.prettierignore';
+    if (ruleArguments.length === 3) {
+      ignorePath = ruleArguments[2];
+    }
+    const r = prettier.getFileInfo.sync(sf!.fileName, { ignorePath });
```

Comment:
```suggestion
    const fileInfo = prettier.getFileInfo.sync(sf!.fileName, { ignorePath });
```

Reply:
No problem, although probably best to just
`return prettier.getFileInfo.sync(sf!.fileName, { ignorePath }).ignored;`

Reply:
I used your suggestion